cvat/apps/engine/models.py

```diff
@@ -857,7 +857,7 @@ class AttributeSpec(models.Model):
     input_type = models.CharField(max_length=16,
         choices=AttributeType.choices())
     default_value = models.CharField(blank=True, max_length=128)
-    values = models.CharField(blank=True, max_length=4096)
+    values = models.CharField(blank=True, max_length=8192)
```

Comment:
Probably, it is not a good solution to increase the limit here and use `CharField`. Since we're using PostgreSQL, there shouldn't be any problems with wasting memory when the `values` field is small, but...  Perhaps it will be better to use `TextField` here and add a limit in the code (but not to the `values` field since we store a list with attribute values in a string with a custom separator) - introduce an upper limit for attribute value and max number of attributes. + This looks logical to me also because we have a limit for `AttributeVal.value`.